data/json/items/grenades.json

```diff
@@ -1,4 +1,185 @@
 [
+  {
+    "id": "grenade",
+    "type": "TOOL",
+    "category": "weapons",
+    "name": "grenade",
+    "description": "This is a military grade fragmentary hand grenade.  Use this item to pull the pin and light the fuse.  You will then have five turns before it explodes; throwing it would be a good idea.",
+    "weight": 397,
+    "volume": 1,
+    "price": 1500,
+    "to_hit": -1,
+    "bashing": 6,
+    "material": [ "steel", "plastic" ],
+    "symbol": "*",
+    "color": "green",
+    "use_action": {
+      "target": "grenade_act",
+      "msg": "You pull the pin on the grenade.",
+      "target_charges": 5,
+      "active": true,
+      "menu_text": "Pull pin",
+      "type": "transform"
+    },
+    "flags": [ "RADIO_MODABLE", "RADIO_INVOKE_PROC", "BOMB", "GRENADE" ]
+  },
+  {
+    "id": "grenade_act",
+    "type": "TOOL",
+    "category": "weapons",
+    "name": "active grenade",
+    "description": "This is an active grenade, and will explode any second now.  Better throw it!",
+    "weight": 397,
+    "volume": 1,
+    "price": 0,
+    "to_hit": -1,
+    "bashing": 6,
+    "material": [ "steel", "plastic" ],
+    "symbol": "*",
+    "color": "green",
+    "initial_charges": 5,
+    "max_charges": 5,
+    "turns_per_charge": 1,
+    "use_action": {
+      "type": "explosion",
+      "sound_volume": 0,
+      "sound_msg": "Tick.",
+      "no_deactivate_msg": "You've already pulled the %s's pin, try throwing it instead.",
+      "explosion": { "power": 185, "shrapnel": 212 }
+    },
+    "flags": [ "BOMB", "TRADER_AVOID" ]
+  },
+  {
+    "id": "makeshift_grenade",
+    "copy-from": "grenade",
+    "type": "TOOL",
+    "name": "makeshift grenade",
+    "description": "An improvised explosive device cobbled together from parts. Use this item to pull the pin and light the fuse.  You will then have some amount of time before it explodes; throwing it would be a good idea.",
```

Comment:
```suggestion
    "description": "An improvised explosive device cobbled together from parts.  Use this item to pull the pin and light the fuse.  You will then have some amount of time before it explodes; throwing it would be a good idea.",
```